runtime/filetype.vim

```diff
@@ -1721,7 +1721,7 @@ au BufNewFile,BufRead *.latex,*.sty,*.dtx,*.ltx,*.bbl	setf tex
 au BufNewFile,BufRead *.tex			call dist#ft#FTtex()
 
 " ConTeXt
-au BufNewFile,BufRead *.mkii,*.mkiv,*.mkvi   setf context
+au BufNewFile,BufRead *.mkii,*.mkiv,*.mkvi,*.mkxl,*.mklx   setf context
```

Comment:
Update the test case for `context` filetype in src/testdir/test_filetype.vim.

Reply:
[Done](https://github.com/champignoom/vim/pull/1/commits/78f8573ade2cf13c3e53500f78745f137c8bd877)